.eslintrc.json

```diff
@@ -11,5 +11,8 @@
 	"env": {
 		"es6": true
 	},
-	"extends": "eslint:recommended"
+	"extends": [
+		"eslint:recommended",
+		"plugin:react/recommended"
+	]
```

Comment:
This change removed this warning 
`Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration`

Reply:
@justin0022 I have put the fix for that warning and it shouldn't be showing you that warning
I have run the `standard --fix` command and it is not showing it for me. 

Here is below

```
13:05:07-pushyami~/src/itsProjects/my-learning-analytics (Formating-issues)$ standard --fix
standard: Use JavaScript Standard Style (https://standardjs.com)
  /Users/pushyami/src/itsProjects/my-learning-analytics/sdenv/share/jupyter/nbextensions/jupyter-js-widgets/extension.js:47:354481: Parsing error: Unexpected token else
13:05:37-pushyami~/src/itsProjects/my-learning-analytics (Formating-issues)$
```
Please ignore the Jupyter-js-widgets waring it is not relevent

Reply:
Ah I just realized I needed to do a docker rebuild - now that I've done so, I can confirm this warning does not display.